src/lib/y2storage/dialogs/guided_setup.rb

```diff
@@ -60,7 +60,9 @@ def dialog_content
               VSpacing(1),
               Left(RadioButton(Id(:mode_lvm), _("LVM-based"), lvm_selected?)),
               VSpacing(1),
-              Left(RadioButton(Id(:mode_encrypted), _("Encrypted LVM-based"), encrypted_selected?))
+              Left(RadioButton(Id(:mode_encrypted), _("Encrypted LVM-based"), encrypted_selected?)),
+              VSpacing(1),
+              Left(Password(Id(:encryption_password), _("Enter encrytion password")))
```

Comment:
typo: encrytion.

While you fix it, maybe it would be nice to indent this a little bit to the right, so it looks to be "inside" the Encrypted LVM-based. I know the dialog is just temporary, but...